src/GovUk.Education.ExploreEducationStatistics.Content.Model.Test/ReleaseTests.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Test
+{
+    public class ReleaseTests
+    {
+        [Fact]
+        public void Release_Not_Live()
+        {
+            {
```

Comment:
Shouldn't need these extra { }.

Might be worth splitting this into two seperate tests

Reply:
Split into two test methods